package.json

```diff
@@ -28,7 +28,7 @@
   },
   "homepage": "https://github.com/Particular/setup-oracle-action#readme",
   "dependencies": {
-    "@actions/core": "^1.9.1",
+    "@actions/core": "^1.10.0",
```

Comment:
@soujay Looks like we're missing the results of doing `npm install` and `npm run prepare`, but on the other hand, the CI log shows no warnings either. Can you double check that doing that doesn't change anything?